.github/workflows/conformance-aks.yaml

```diff
@@ -15,13 +15,29 @@ on:
   ###
 
 concurrency:
-  # In case of PR comment, we can't simply append the comment to the group name
-  # as that is too long and results in an error. Instead we check if it's a
-  # trigger phrase and append a simple 'trigger-phrase'.
+  # Structure:
+  # - Workflow name
+  # - Event type
+  # - A unique identifier depending on event type:
+  #   - schedule: SHA
+  #   - issue_comment: PR number + trigger phrase
+  #   - pull_request: PR number + label name
+  #
+  # This structure ensures a unique concurrency group name is generated for each type of testing:
+  # - schedule: {name} schedule {SHA}
+  # - issue_comment: {name} issue_comment {PR number} {trigger phrase}
+  # - pull_request: {name} pull_request {PR number} {label name}
   group: |
-    "${{ github.workflow }} ${{ github.event.issue.pull_request.url || 'scheduled' }}
-     ${{ (startsWith(github.event.comment.body, 'ci-aks') ||
-          startsWith(github.event.comment.body, 'test-me-please')) && 'trigger-phrase' }}"
+    ${{ github.workflow }}
+    ${{ github.event_name }}
+    ${{ github.event.issue.number || github.event.pull_request.number || github.sha }}
+    ${{ github.event_name == 'issue_comment' && (
+      startsWith(github.event.comment.body, 'ci-aks') ||
+      startsWith(github.event.comment.body, 'test-me-please')
+    ) && 'ci-aks' }}
+    ${{ github.event_name == 'pull_request'
+      && github.event.label.name == 'ci-run/aks'
```

Comment:
I assume we'll need to drop this part now that we're getting rid of labels.

Reply:
Indeed! Will make changes accordingly.

Reply:
Fixed, PTAL.